Core/Utilities/Time.py

```diff
@@ -236,6 +236,17 @@ def includes( self, myDateTime ):
       return False
     return True
 
+def queryTime(f):
+  """ Decorator to measure the function call time
+  """
+  def measureQueryTime(*args, **kwargs):
+    start = time.time()
+    result = f(*args, **kwargs)
+    if not 'QueryTime' in result:
```

Comment:
I would say that this is rather DError is incompatible. We have in many places mixing in some additional information to the return dictionary. I guess DError will screw up those cases